src/components/Repeat/RepeatForm/RepeatForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import Input from 'components/Shared/Form/Input';
+import ButtonIcon from 'components/Shared/Buttons/ButtonIcon';
+import * as S from 'pages/User/Learning/Repeat/StyleRepeat';
+import { TitleBig } from 'components/Shared/Atoms/Title';
+
+interface RepeatFormProps {
+  daysRepeat: number;
+  onDaysRepeatChange: (value: number) => void;
+  onSubmit: () => void;
+}
+
+const RepeatForm: FC<RepeatFormProps> = ({ daysRepeat, onDaysRepeatChange, onSubmit }) => {
+  const handleDaysRepeatChange = (value: string) => {
+    const numericValue = parseInt(value, 10);
```

Comment:
`const numericValue = parseInt(value, 10);`
zamiast tego możesz też uzyć `+value`. Ale to juz jak chcesz. bez różnicy :)